core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSuggestionRequest.java

```diff
@@ -68,7 +70,7 @@
 	@Min(1)
 	private int minOccurrenceCount;
 	
-	private transient List<String> topTokens;
+	private transient List<String> topTokens = Collections.emptyList();
```

Comment:
I think the `emptyList()` assignment is not needed,

Reply:
You are right, it's not needed 🎉